src/status_im/translations/ja.cljs

```diff
@@ -158,83 +177,204 @@
+   :postponed                             "延期された"
+
+   ;;webview
+   :web-view-error                         "おっと、エラーが発生しました。"
+
+   ;;testfairy warning
+   :testfairy-title                       "警告!"
+   :testfairy-message                     "あなたはインストールされたアプリは夜間のバージョンです。 テスト目的のために、このバージョンにWi-Fi接続が使用されている場合は会話記録が含まれています。このアプリとのすべてのやり取りは（動画やログとして）保存され、開発チームが問題の調査に使用する可能性があります。 保存されたビデオ/ログにはパスワードは含まれません。 会話記録は、アプリが夜間のバージョンからインストールされている場合にのみ実行されます。 アプリがPlayStoreまたはTestFlightからインストールされている場合は記録されません。"
+
+   ;; wallet
+   :wallet                                "財布"
+   :wallets                               "全ての財布"
+   :your-wallets                          "あなたの全ての財布"
+   :main-wallet                           "メインウォレット"
+   :wallet-error                          "データの読み込みエラー"
+   :wallet-send                           "送信"
+   :wallet-send-token                     "送信 {{symbol}}"
+   :wallet-request                        "要求"
+   :wallet-exchange                       "交換"
+   :wallet-assets                         "資産"
+   :wallet-add-asset                      "資産を追加"
+   :wallet-total-value                    "総価値"
+   :wallet-settings                       "ウォレット設定"
+   :wallet-manage-assets                  "資産の管理"
+   :signing-phrase-description            "パスワードを入力して取引に署名します。 上記の単語があなたの秘密の署名句に一致することを確認してください"
+   :wallet-insufficient-funds             "残高不足"
+   :wallet-my-token                       "わたしの {{symbol}}"
+   :wallet-market-value                   "市場価値"
+   :request-transaction                   "取引をリクエストする"
+   :send-request                          "リクエストを送信する"
+   :share                                 "シェア"
+   :eth                                   "ETH"
+   :currency                              "通貨"
+   :usd-currency                          "USD"
+   :transactions                          "取引"
+   :transaction-details                   "取引の詳細"
+   :transaction-failed                    "取引が失敗しました"
+   :transactions-sign                     "サイン"
+   :transactions-sign-all                 "すべてにサインする"
+   :transactions-sign-transaction         "取引に署名する"
+   :transactions-sign-later               "後でサインする"
+   :transactions-delete                   "取引を削除する"
+   :transactions-delete-content           "取引は「未署名」リストから削除されます。"
+   :transactions-history                  "歴史"
+   :transactions-unsigned                 "署名なし"
+   :transactions-history-empty            "あなたの履歴にまだ取引はありません"
+   :transactions-unsigned-empty           "署名のない取引はありません"
+   :transactions-filter-title             "フィルタ履歴"
+   :transactions-filter-tokens            "トークン"
+   :transactions-filter-type              "タイプ"
+   :transactions-filter-select-all        "全て選択"
+   :view-transaction-details              "取引の詳細を表示する"
+   :transaction-description               "あなたの取引が安全に処理されることを確認するために少なくとも12回の確認を待ってください。"
+   :transaction-sent                      "送信された取引"
+   :transaction-moved-text                "取引は "未署名"リストに５分間を残ります"
```

Comment:
Change the quoted text to use `'` single quotes, otherwise it breaks everything.